examples/container/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "container-scaleway-starter",
+    "version": "1.0.0",
+    "main": "index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {},
+    "devDependencies": {
+      "serverless-scaleway-functions": "^0.1"
```

Comment:
update this dependency to match current version of serverless framework, + increment version inside your main `package.json` file please for next release with containers